### ISSUE TITLE
'Draw a box' - Page behaving as an image when trying to draw the box

### ISSUE THREAD

**Issue by daZebra** [rsmithlal/ClimateDataRescue#13]
**Issue**
As you can see below, the ledger page behaves as an image when comes the time to draw a box around the transcribed area. I am clicking and dragging, but I am pulling the image instead of making a rectangle.

![cant-drag](https://cloud.githubusercontent.com/assets/14224342/18057637/cbef5c9a-6dde-11e6-8326-3ee28c75552b.gif)

**Comment by rsmithlal** [comment]
Oh boy, that's a small little gif. Can you try doing this again with your javascript console open? on chrome it's right-click and "inspect". Let me know if you get any error messages. Can you provide more info on what conditions lead up to this happening? I see that you have a couple annotations there that were drawn previously.

**Comment by daZebra** [comment]
I sent you the video by Slack. 
I was able to repeat the issue today. Here's what I did:
1 - I made an annotation.
2 - Drew a box.
3 - Resubmitted an identical annotation, for the same field, with the same time and date.
4 - It says 'Annotation saved successfully...", the standard cursor changes into the 'crosshair-cursor', but when I click and drag it behaves as an image that I am dragging across the screen.
